package.json

```diff
@@ -5,9 +5,8 @@
   "main": "devicon.min.css",
   "scripts": {
     "build-css": "gulp updateCss && gulp clean",
-    "peek-test": "python ./.github/scripts/icomoon_peek.py ./.github/scripts/build_assets/geckodriver-v0.27.0-win64/geckodriver.exe ./icomoon.json ./devicon.json ./icons ./  --pr_title \"%PR_TITLE%\"",
-    "build-test": "python ./.github/scripts/icomoon_build.py ./.github/scripts/build_assets/geckodriver-v0.27.0-win64/geckodriver.exe ./icomoon.json ./devicon.json ./icons ./",
-    "optimize-svg": "gulp optimizeSvg",
```

Comment:
Why was `optimize-svg` removed?

Reply:
Whoops, removed that by accident. I wanted to remove `build-test`, as that one doesn't work. It asks for a GitHub token. PR #1668 adds a command to build the icons locally/Gitpod.